### PR TITLE
report resource kinds that are expanded in tree to telemetry (#475)

### LINKED ISSUE #475: report expanding tree categories to telemetry
When trying to find out how much users use our tooling we have telemetry in all actions currently. We're missing telemetry though where users would simply expand categories in the resource tree

### PR DESCRIPTION
**Pull request by adietish**
fixes #475

### CODE REVIEW

File: src/main/kotlin/com/redhat/devtools/intellij/kubernetes/tree/ResourceWatchController.kt [hunk 1]
```diff
@@ -26,18 +28,27 @@ object ResourceWatchController {
 		tree.addTreeExpansionListener(object: TreeExpansionListener {
 			override fun treeExpanded(event: TreeExpansionEvent) {
 				val descriptor = getDescriptor(event)
+				val telementry = createTelemetry(descriptor)
```
Comment (jeffmaury) [inline src/main/kotlin/com/redhat/devtools/intellij/kubernetes/tree/ResourceWatchController.kt:31]: Rename as telemetry

**Review by jeffmaury — CHANGES_REQUESTED** [review]

File: src/main/kotlin/com/redhat/devtools/intellij/kubernetes/tree/ResourceWatchController.kt (hunk 1 above)
Comment (adietish) [reply src/main/kotlin/com/redhat/devtools/intellij/kubernetes/tree/ResourceWatchController.kt]: corrected the spelling error "telementry" -> "telemetry", thanks for spotting.

File: src/main/kotlin/icons/Icons.kt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package icons
+
+import com.intellij.ui.IconManager
+import javax.swing.Icon
+
+class Icons {
+
+    @JvmField
+    public val terminal = loadIcon("icons/terminal.svg", (hashCode() + "terminal".hashCode()).toLong(), 2)
```
Comment (jeffmaury) [inline src/main/kotlin/icons/Icons.kt:9]: I don't understand the purpose of this class ? As it's new, where it's been used in the code ?

Comment (adietish) [reply src/main/kotlin/icons/Icons.kt:9]: oh, was added by accident. Have to remove it.

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**